src/renderer/zbay/transit.js

```diff
@@ -127,7 +127,7 @@ export const checkMessageSizeAfterComporession = async message => {
 //   return allocatedMessage.toString()
 // }
 
-export const packMemo = async (message, typingIndicato) => {
+export const packMemo = async (message, typingIndicato, onionAddressProp, zcashAddressProp) => {
```

Comment:
instead of passing empty string in every function call, better use default parameters e.g onionAddressProp = ''